src/components/ChartControls.jsx

```diff
@@ -12,13 +12,13 @@ import Share from './Share.jsx';
 import '../../sass/components/_chart-controls.scss';
 
 
-export const RenderDefaultControls = ({ isMobile }) => (
+export const RenderDefaultControls = ({ isMobile, isHistorical }) => (
     <>
         {isMobile ? '' : <CrosshairToggle />}
         <ChartTypes />
         <Timeperiod />
         <StudyLegend />
-        <Comparison />
+        {isHistorical ? '' : <Comparison />}
```

Comment:
is this suppose to disable comparisons when historical mode is on? It's not doing so. The controls are rendered in the app code. In fact, you should not put this logic in the library since the library doesn't know there's something called historical mode. It's just a switch in the settings.